consensus/mint-client/src/config.rs

```diff
@@ -349,11 +348,11 @@ pub enum Commands {
     /// Sign governors configuration from a tokens.toml/tokens.json file.
     SignGovernors {
         /// The key to sign with.
-        #[clap(long = "signing-key", parse(try_from_str = load_key_from_pem), env = "MC_MINTING_SIGNING_KEY")]
+        #[clap(long = "signing-key", value_parser = load_key_from_pem, env = "MC_MINTING_SIGNING_KEY")]
         signing_key: Ed25519Private,
 
         /// The tokens configuration file to sign (in JSON or TOML format).
-        #[clap(long, parse(try_from_str = TokensConfig::load_from_path), env = "MC_MINTING_TOKENS_CONFIG")]
+        #[clap(long, value_parser = parse_tokens_file, env = "MC_MINTING_TOKENS_CONFIG")]
```

Comment:
It no longer lets you access static methods in a struct? :(

Reply:
It's not static methods it's lifetimes, which could be my rust ignorance on a better solution
```shell
error[E0308]: mismatched types
   --> consensus/mint-client/src/config.rs:373:22
    |
373 |         #[clap(long, value_parser = TokensConfig::load_from_path, env = "MC_MINTING_TOKENS_CONFIG")]
    |                      ^^^^^^^^^^^^ lifetime mismatch
    |
    = note: expected type `for<'r> Fn<(&'r str,)>`
               found type `Fn<(&str,)>`
note: the lifetime requirement is introduced here
   --> /home/nick/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-4.0.10/src/builder/arg.rs:943:48
    |
943 |     pub fn value_parser(mut self, parser: impl IntoResettable<super::ValueParser>) -> Self {
    |     
```

Reply:
Hm, That is pretty weird 🤷 but not worth spending time on